src/ranged.cpp

```diff
@@ -1874,15 +1871,7 @@ dispersion_sources Character::get_weapon_dispersion( const item &obj ) const
     dispersion_sources dispersion( weapon_dispersion );
     dispersion.add_range( ranged_dex_mod() );
 
-    int arm_encumb = 0;
-    // Fake turret NPC does not have arms
-    if( has_part( bodypart_id( "arm_l" ) ) ) {
-        arm_encumb += encumb( bodypart_id( "arm_l" ) );
-    }
-    if( has_part( bodypart_id( "arm_r" ) ) ) {
-        arm_encumb += encumb( bodypart_id( "arm_r" ) );
-    }
-    dispersion.add_range( arm_encumb / 5.0 );
+    dispersion.add_range( ranged_dispersion_modifier() );
```

Comment:
I trust this "fake turret NPC" will not be a difficulty for the body part code?

Reply:
nah. if it's an npc it'll have arms and legs and it'll probably end up being 1.0 across the board. i'll admit i haven't looked into it specifically though.